pkg/analysis/definition.go

```diff
@@ -0,0 +1,31 @@
+package analysis
+
+import (
+	"context"
+
+	"go.lsp.dev/protocol"
+
+	"github.com/tilt-dev/starlark-lsp/pkg/document"
+	"github.com/tilt-dev/starlark-lsp/pkg/query"
+)
+
+func (a *Analyzer) Definition(ctx context.Context, doc document.Document, pos protocol.Position) ([]protocol.Location, error) {
```

Comment:
Noticed we don't have any cases that return an error - do you anticipate those in the future, or is it worth removing the unused error return param?